docs/conf.py

```diff
@@ -23,7 +23,7 @@
 author = 'Developer Relations'
 
 # The full version, including alpha/beta/rc tags
-release = '1.0a'
+release = '1.0.0'
```

Comment:
file docs/changelog.rst need updating to list the new APIs added

Reply:
done